src/index.js

```diff
@@ -272,10 +272,10 @@ const forms = plugin.withOptions(function (options = { strategy: undefined }) {
       },
       {
         base: [
-          `[type='checkbox']:checked:hover`,
-          `[type='checkbox']:checked:focus`,
-          `[type='radio']:checked:hover`,
-          `[type='radio']:checked:focus`,
+          `input:where([type='checkbox']):checked:hover`,
+          `input:where([type='checkbox']):checked:focus`,
+          `input:where([type='radio']):checked:hover`,
+          `input:where([type='radio']):checked:focus`,
```

Comment:
@adamwathan Should we put these pseudo classes inside `:where(…)`? I'm not sure.